package.json

```diff
@@ -310,7 +311,8 @@
     "url-loader": "^4.1.1",
     "web-encoding": "^1.1.5",
     "webpack-manifest-plugin": "^2.2.0",
-    "workbox-webpack-plugin": "^4.3.1"
+    "workbox-webpack-plugin": "^4.3.1",
+    "write-file-webpack-plugin": "^4.5.1"
   },
   "resolutions": {
     "hoist-non-react-statics": "^3.3.0"
```

Comment:
please restore the following 3 image files (one gif and two svgs)
there are **our custom** images. they should be kept in **our** repo
webpack copy plugin should copy "stock" images keeping those files untouched